package.json

```
@@ -291,7 +291,8 @@
 			"ci:build:docs": {
 				"dependsOn": [
 					"^ci:build:docs",
-					"build:gendocs:client"
+					"build:gendocs:client",
+					"ci:build:readme"
```

There's no ci:build:readme task, so this should be:

```suggestion
					"build:readme"
```

---

Autopilot... 🤦‍♂️